src/main/java/org/hisp/dhis/tasks/analytics/LoadDashboardTask.java

```diff
@@ -35,19 +37,23 @@ public String getType()
     public void execute()
         throws Exception
     {
-        User user = getUser();
+        Randomizer rnd = getNextRandomizer( getName() );
+        User user = getUser( rnd );
 
-        Dashboard dashboard = DataRandomizer.randomElementFromList( entitiesCache.getDashboards() );
+        Dashboard dashboard = rnd.randomElementFromList( entitiesCache.getDashboards() );
 
         // app loads items as user scrolls, so load 10 at once
-        List<Integer> randomSequence = DataRandomizer.randomSequence( dashboard.getDashboardItems().size(), 10 );
+        List<Integer> randomSequence = IntStream.range(0, dashboard.getDashboardItems().size())
```

Comment:
If I got it right 😋 this might be easier to read as the range and then the mapping ignoring the argument threw me off at first.

```java
List<Integer> randomSequence = IntStream
          .generate(() -> rnd.randomInt(10))
          .boxed()
          .limit(dashboard.getDashboardItems().size())
          .collect(Collectors.toList());
```

Since this is already a stream we could also directly call `parallel()` on it and merge it with the code below.